airflow/jobs/scheduler_job.py

```diff
@@ -1557,6 +1560,10 @@ def _process_executor_events(self, simple_dag_bag: SimpleDagBag, session: Sessio
             state, info = event_buffer.pop(buffer_key)
 
             # TODO: should we fail RUNNING as well, as we do in Backfills?
```

Comment:
Not related, but is this comment still valid?

Reply:
Not sure off the top of my head.